From c88e6ed877a72e21a39271ea851589523326af2c Mon Sep 17 00:00:00 2001
From: Hans Verkuil <hverkuil@xs4all.nl>
Date: Mon, 7 Aug 2017 14:52:01 +0200
Subject: keytable.c: add support for the CEC protocol

The CEC protocol wasn't known, so 'Supported protocols:' would just say
'other' instead of 'cec'.

Signed-off-by: Hans Verkuil <hans.verkuil@cisco.com>
Acked-by: Sean Young <sean@mess.org>

diff --git a/utils/keytable/keytable.c b/utils/keytable/keytable.c
index 634f456..55abfc1 100644
--- a/utils/keytable/keytable.c
+++ b/utils/keytable/keytable.c
@@ -106,6 +106,7 @@ enum sysfs_protocols {
 	SYSFS_RC6		= (1 << 10),
 	SYSFS_SHARP		= (1 << 11),
 	SYSFS_XMP		= (1 << 12),
+	SYSFS_CEC		= (1 << 13),
 	SYSFS_INVALID		= 0,
 };
 
@@ -138,6 +139,7 @@ const struct protocol_map_entry protocol_map[] = {
 	{ "rc-6-mce",	NULL,		SYSFS_INVALID	},
 	{ "sharp",	NULL,		SYSFS_SHARP	},
 	{ "xmp",	"/xmp_decoder",	SYSFS_XMP	},
+	{ "cec",	NULL,		SYSFS_CEC	},
 	{ NULL,		NULL,		SYSFS_INVALID	},
 };
 
-- 
cgit v0.10.2

